tests/Doctrine/Tests/Common/Annotations/AnnotationRegistryTest.php

```diff
@@ -65,4 +64,107 @@ protected function getStaticField($class, $field)
 
         return $reflection->getValue();
     }
+
+    /**
+     * @runInSeparateProcess
+     */
+    public function testStopCallingLoadersIfClassIsNotFound() : void
+    {
+        AnnotationRegistry::reset();
+        $i = 0;
+        $autoLoader = function($annotation) use (&$i) {
+            $i++;
```

Comment:
`++$i`

Reply:
Why? Just your preference or some standard or reason?

I know this is not Symfony, but https://cs.sensiolabs.org/ changed all my incrementations from `$i++` and `$i += 1` to `++$i`.

Reply:
This was discussed under https://github.com/doctrine/coding-standard/pull/87 - we can go with `$i++`.